privacy_partner_to_be_forgotten/readme/USAGE.md

```
@@ -0,0 +1,17 @@
+## To anonymize a partner record:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)